docs/docs/source-plugin-tutorial.md

```diff
@@ -355,7 +355,14 @@ Experiment with different options in your `gatsby-config.js` file to see how tha
 
 You've built a local plugin for your project, but what if you want to share it with others? `npm` allows you to publish your plugins. Check out the npm docs on [How to Publish & Update a Package](https://docs.npmjs.com/getting-started/publishing-npm-packages) for more info.
 
-> **NOTE:** Don't forget to edit your plugin's `package.json` file to include info about your plugin.
+> **NOTE:** Don't forget to edit your plugin's `package.json` file to include info about your plugin. To make sure it is included in the  [Gatsby Plugin Library](https://www.gatsbyjs.org/plugins/), add a `keywords` field to package.json containing `gatsby` and `gatsby-plugin`.
```

Comment:
Maybe we could link to https://www.gatsbyjs.org/docs/plugin-authoring/#publishing-a-plugin-to-the-library instead of duplicating content?

Reply:
Great catch @conradj -- it makes sense to include something about publishing plugins to Gatsby in this tutorial. However, I would say it's important actually to also say something like: 

"Don't publish this particular plugin to npm or the Gatsby Plugin Library, because it's just a sample plugin for the tutorial. However, if you create another source plugin and would like to publish it to npm, follow the npm docs on [How to Publish & Update a Package](https://docs.npmjs.com/getting-started/publishing-npm-packages) for more info.
		
If you'd like to publish a plugin to the [Gatsby Plugin Library](/packages/) (please do!), please follow the [Plugin Authoring guide](/docs/plugin-authoring/).